src/store/aplication/aplicationAction.js

```diff
@@ -51,7 +51,7 @@ export const restoreAccountWithPin = (pin, callback) => async (dispatch) => {
   const shaPing = await bitcoin.sha256(pin);
   database.restoreWithPin(shaPing).then(async (data) => {
     bitcoin.createWallet(data.seed[0].seed);
-    await chatService.startService();
+    // await chatService.startService();
```

Comment:
Shouldn't this be removed?

Reply:
this should not be  removed,  was commented for  testing of the device settings component